docs/source/tutorials/configuration.rst

```diff
@@ -353,12 +353,12 @@ A list of all config options
    'frame_y_radius', 'from_animation_number', 'images_dir', 'input_file',
    'leave_progress_bars', 'left_side', 'log_dir', 'log_to_file',
    'max_files_cached', 'media_dir', 'media_width', 'movie_file_extension', 'output_file',
-   'partial_movie_dir', 'pixel_height', 'pixel_width', 'plugins', 'png_mode',
+   'partial_movie_dir', 'pixel_height', 'pixel_width', 'plugins', 'png_mode', 
    'preview', 'progress_bar', 'quality', 'right_side', 'save_as_gif', 'save_last_frame',
-   'save_pngs', 'scene_names', 'show_in_file_browser', 'sound', 'tex_dir',
+   'save_pngs', 'scene_names', 'show_in_file_browser', 'sound', 'streaming_dir', 'tex_dir', 
    'tex_template', 'tex_template_file', 'text_dir', 'top', 'transparent',
-   'upto_animation_number', 'use_opengl_renderer', 'use_webgl_renderer', 'verbosity',
-   'video_dir', 'webgl_renderer_path', 'webgl_updater_fps', 'write_all',
+   'upto_animation_number', 'use_opengl_renderer', 'use_webgl_renderer', 'verbosity', 
+   'video_dir', 'webgl_renderer_path', 'webgl_updater_fps', 'write_all', 
```

Comment:
As of the click update, this section is no longer being doctested due to popular request, but feel free to update it here anyways.